docs/server-side/v2/introduction/webhook.mdx

```diff
@@ -569,18 +569,20 @@ This event will be sent when final composed recording is generated and uploaded
-| session_id         | `string`             | 100ms assigned id to identify the session <br/><br/> _Example: 5f9edc6bd238215aec7700df _                                          |
-| template_id        | `string`             | Template ID of the room <br/><br/> _Example: 66112497abcd52312556c4gg _                                                            |
-| location           | `string`             | s3 URI of the recorded video <br/><br/> _Example: s3://bucket/prefix/ac.mp4 _                                                      |
-| URL                | `string`             | Presigned s3 URL for the recorded video, for download. Valid for 24 hours <br/><br/> _Example: https://s3-location/bucket/ac.mp4 _ |
-| duration           | `number`             | Duration the user spent in the room in seconds <br/><br/> _Example: 3600 _                                                         |
-| size               | `number`             | Size of the recorded video (in bytes) <br/><br/> _Example: 10024 _                                                                 |
-| session_started_at | `timestamp (in UTC)` | Timestamp when session started <br/><br/> _Example: 2020-11-11T16:32:17Z _                                                         |
-| session_stopped_at | `timestamp (in UTC)` | Timestamp when session ended <br/><br/> _Example: 2020-11-11T16:32:17Z _                 |
+| Name                    | Type                 | Description                                                                                                                        |
+| :---------------------- | :------------------- | :--------------------------------------------------------------------------------------------------------------------------------- |
+| room_id                 | `string`             | 100ms assigned room id <br/><br/> _Example: 5f9edc6ac238215aec2312df _                                                             |
+| room_name               | `string`             | Room name assigned by you when creating room <br/><br/> _Example: class-9-batch-2 _                                                |
+| session_id              | `string`             | 100ms assigned id to identify the session <br/><br/> _Example: 5f9edc6bd238215aec7700df _                                          |
+| template_id             | `string`             | Template ID of the room <br/><br/> _Example: 66112497abcd52312556c4gg _                                                            |
+| location                | `string`             | s3 URI of the recorded video <br/><br/> _Example: s3://bucket/prefix/ac.mp4 _                                                      |
+| URL                     | `string`             | Presigned s3 URL for the recorded video, for download. Valid for 24 hours <br/><br/> _Example: https://s3-location/bucket/ac.mp4 _ |
```

Comment:
@chdinesh1089 Let's add a note that this is deprecated and recommend using recording_presigned_url. Similarly in other applicable instances. 
CC: @sagargaonkar

Reply:
Made the changes.